ihm/multi_state_scheme.py

```diff
@@ -0,0 +1,225 @@
+# coding=utf-8
+
+"""Classes for handling connected/ordered schemes formed by multiple state
+    together with information on kinetic schemes"""
+import ihm.multi_state_scheme
+
+
+class MultiStateScheme(object):
+    """MultiStateScheme collects information about a collection of
+       multiple states, that can form a connected/ordered scheme.
+       A special case is a kinetic scheme, for which kinetic rates and
+       relaxation times are available.
+
+       :param str name: The name of the multi-state scheme.
+       :param str details: Details on the scheme.
+       :param list_of_connectivities: A list of connectivities that belong to
```

Comment:
I think this can just be "connectivities". The `list_of` prefix is unnecessarily verbose.